### ISSUE TITLE
Where with RAW doesn't work properly 

### ISSUE THREAD

**Issue by mrcnpdlk** [skipperbent/pecee-pixie#137]
Changes made for version [4.12.1](https://github.com/skipperbent/pecee-pixie/compare/4.12.0...4.12.1) crashed Raw support for where clause.
For Raw object method [buildCriteria()](https://github.com/skipperbent/pecee-pixie/blob/e604f433f3f9346f83a4767870a10f360facf93f/src/Pecee/Pixie/QueryBuilder/Adapters/BaseAdapter.php#L116) adds the table prefix now.
For [example](https://github.com/skipperbent/pecee-pixie#raw-expressions) from documentation the library throws the exception: `Unknown column 'someALiasTable.DATE()' in 'where clause` if the alias is specified.

**Comment by mrcnpdlk** [comment]
In this [line ](https://github.com/skipperbent/pecee-pixie/blob/e604f433f3f9346f83a4767870a10f360facf93f/src/Pecee/Pixie/QueryBuilder/Adapters/BaseAdapter.php#L140) it should be checked if the `key` is `Raw` instance.

**Comment by skipperbent** [comment]
Hi Marcin.

Thanks for pointing out the issue and the very detailed description. Helped a lot!
I was on vacation last week and the laptop was so old that it would take forever to update everything (php, phpstorm etc) to fix the issue at that point. Now I'm back and I've created a patch that will be released shortly. I've also added some unit-tests to *hopefully* catch this type of issue in the future :) 

/ Simon
